deploy/MangoHost.ts

```diff
@@ -37,7 +44,9 @@ const deployFunction: DeployFunction = async function (
       },
     },
     log: true,
+    gasPrice,
   })
+  console.log('After Eth balance', await getEthBalance(deployer))
```

Comment:
Could you change `console.log` to `liveLog`?

Reply:
nice I've fixed it : https://github.com/clober-dex/mango-farmers-club/pull/12/commits/3e2409a75da15c4a44f8acb6958f5020e4b74d0e @JhChoy